themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -0,0 +1,9 @@
+@import '~terra-mixins/lib/Mixins';
```

Comment:
This file can be removed.

Reply:
Removed it https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8